must-gather/collection-scripts/gather_selinux

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+. namespace
+NRO_NAMESPACE=$(nro_namespace)
+
+SELINUX_INFO_DIR="/must-gather/selinux_info"
+mkdir -p "$SELINUX_INFO_DIR"
+
+# common variables
+OC_GET_PODS="oc get pods -n $NRO_NAMESPACE"
+
+function replace_dot_with_underscore() {
+    echo $1 | sed 's/\./_/g'
+}
+
+function gather_selinux_data() {
+    local rte_pods=$($OC_GET_PODS -l name='resource-topology' -o jsonpath='{.items[*].metadata.name}')
+    
+    for pod in $rte_pods; do
+        local original_node_name=$($OC_GET_PODS $pod -o jsonpath='{.spec.nodeName}')
+        local node_name=$(replace_dot_with_underscore $original_node_name)
+        
+        echo "Gathering SELinux data from node: $node_name"
+        
+        local node_dir="$SELINUX_INFO_DIR/$node_name"
+        mkdir -p "$node_dir"
+        
+        oc debug node/$original_node_name -- bash -c "
+            chroot /host bash -c '
+                mkdir -p /tmp/selinux_data
+                
+                echo \"=== SELinux context for /var/lib/kubelet ===\" > /tmp/selinux_data/contexts
+                ls -Z /var/lib/kubelet >> /tmp/selinux_data/contexts 2>&1
+                echo \"\" >> /tmp/selinux_data/contexts
+                echo \"=== SELinux context for kubelet.sock ===\" >> /tmp/selinux_data/contexts
+                ls -Z /var/lib/kubelet/pod-resources/kubelet.sock >> /tmp/selinux_data/contexts 2>&1
```

Comment:
if must-gather ran more than once some data will be appended (>>) to what exists, that isn't what we want, is it?

Reply:
no problem I missed that on line 32 you are overriding what's in contexts file